onnxruntime/python/tools/tensorrt/perf/post.py

```diff
@@ -84,10 +80,12 @@ def get_status(status, model_group):
     return status
 
 def get_specs(specs, branch, commit_id, date_time):
-    specs = specs.append({'.': 6, 'Spec': 'Branch', 'Version' : branch}, ignore_index=True)
-    specs = specs.append({'.': 7, 'Spec': 'CommitId', 'Version' : commit_id}, ignore_index=True)
-    specs = specs.append({'.': 8, 'Spec': 'CommitTime', 'Version' : date_time}, ignore_index=True)
-    return specs
+    init_id = int(specs.tail(1).get('.', 0)) + 1
+    specs_additional = pd.DataFrame({'.': [init_id, init_id + 1, init_id + 2],
+                                     'Spec': ['Branch', 'CommitId', 'CommitTime'],
+                                     'Version': [branch, commit_id, date_time]})
+
+    return pd.concat([specs, specs_additional], ignore_index=True)
```

Comment:
I temporarily modified post.py to print the resulting specs dataframe, which is shown below.
```shell
   .               Spec                                            Version
0  1                CPU  Model name:  Intel(R) Xeon(R) Silver 4114 CPU ...
1  2                GPU     NVIDIA Corporation TU104GL [Tesla T4] (rev a1)
2  3           TensorRT  tensorrt_version_8_2_1_8 , *All ORT-TRT and TR...
3  4               CUDA                                 CUDA Version: 11.5
4  5              CuDNN                                              8.3.1
5  6  EPOptionOverrides  {"TensorrtExecutionProvider": {"trt_engine_cac...
6  7             Branch                         adrianl/py-perf-ep-options
7  8           CommitId                                          b47e5f95c
8  9         CommitTime                                         2022-04-20
```